mobile_files/package-lock.json

```diff
@@ -3520,6 +3520,468 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
+    "fsevents": {
```

Comment:
i'm not quite sure why anything was added here, module was removed via `npm uninstall nfc-react-native`